opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -220,9 +219,10 @@ message Span {
   // Status.Ok (code = 0).
   Status status = 16;
 
-  // An optional number of child spans that were generated while this span
-  // was active. If set, allows an implementation to detect missing child spans.
-  google.protobuf.UInt32Value child_span_count = 17;
+  // An optional number of local child spans that were generated while this span
+  // was active. Value of -1 indicates that the number of local child spans is unknown.
+  // If local_child_span_count>=0, allows an implementation to detect missing child spans.
+  sfixed32 local_child_span_count = 17;
```

Comment:
Comment on the Span ` The next field id is 18.` have to be updated

Reply:
I think the comment is correct. It means the next _available_. If it is unclear I can fix it.

Reply:
Oh, I totally missed that this is not a new property. Sorry for confusion